src/base/projects/Commit/CommitAuthorship.svelte

```diff
@@ -1,16 +1,20 @@
 <script lang="ts">
   import type { CommitMetadata } from "@app/commit";
+
+  import Badge from "@app/Badge.svelte";
   import { formatTimestamp, gravatarURL } from "@app/utils";
 
   export let commit: CommitMetadata;
-  export let showTime = true;
-  export let showAuthor = true;
+  export let noTime = false;
+  export let noAuthor = false;
+  export let noDelegate = false;
```

Comment:
Not a huge deal, but my preference would be to leave these the way they were, because it makes the resulting code easier to read by avoiding double negatives.

E.g. `{#if !noAuthor}` I read in my mind as "no no author", is a bit more mental strain than `{#if !showAuthor}` which i read as "don't show author".

Reply:
Alright. We'll keep it.